src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/util/HttpCacheUtils.java

```diff
@@ -290,4 +291,14 @@ public static boolean isPrivateCacheableResponse(ListFW<HttpHeaderFW> responseHe
             return false;
         });
     }
+
+    public static boolean canStore(ListFW<HttpHeaderFW> responseHeaders){
+        if (responseHeaders.anyMatch(h ->
+                "cache-control".equals(h.name().asString())
+                        && h.value().asString().contains(NO_STORE)))
+        {
+            return false;
+        }
+        return true;
+    }
 }
```

Comment:
Suggest implementing a `static` method on `HttpCacheUtils` such as:

```
private static final boolean isCacheControlNoStore(
  HttpHeaderFW header)
{
    final String name = header.name().asString();
    final String value = header.value().asString();
    return HttpHeaders.CACHE_CONTROL.equals(name) && value.contains(NO_STORE))
}
```

...then from the call site using:
```
responseHeaders.anyMatch(HttpCacheUtils::isCacheControlNoStore);
```

Reply:
This is cleaner than my suggestion